cas/cas-server/src/main/resources/overriden_messages.properties

```
@@ -4,7 +4,7 @@ cas.authentication.email.forgot=Vous avez oublié votre identifiant ? Contactez
 cas.authentication.email.input=Veuillez saisir votre adresse mail
 cas.authentication.email.placeholder=Adresse e-mail *
 cas.authentication.password.confirm=Confirmer le nouveau mot de passe *
-cas.authentication.password.create=CRÉER UN NOUVEAU MOT DE PASSE.
+cas.authentication.password.create=Créer un nouveau mot de passe
```

Sur la maquette, c'est avec un "z" :
```suggestion
cas.authentication.password.create=Créez un nouveau mot de passe
```

---

Modifié